doc/release-notes-16528.md

```diff
@@ -0,0 +1,108 @@
+is available. Additionally there may be some bugs and current functions may change in the future.
+
+0.21 introduces a new type of wallet - Descriptor Wallets. Descriptor Wallets store
+scriptPubKey information using descriptors. This is in contrast to the Legacy Wallet
+structure where keys are used to generate scriptPubKeys and addresses. Because of this
+shift to being script based instead of key based, many of the confusing things that legacy
+wallets are not possible with Descriptor Wallets. Descriptor Wallets determine use a definition
+of "mine" for scripts which is simpler and more intuitive than that used by Legacy Wallets.
+Descriptor Wallets also uses different semantics for watch-only things and imports.
+
+As Descriptor Wallets are a new type of wallet, their introduction does not effect existing wallets.
+Users who already have a Bitcoin Core wallet can continue to use it as they did before without
+any change in behavior. Newly created Legacy Wallets (which is the default type of wallet) will
+behave as they did in previous versions of Bitcoin Core.
+
+For users who decide to use Descriptor Wallets, the differences between them and Legacy Wallets
+is largely limited to non user facing things. For most users, Descriptor Wallets will behave
+similarly to Legacy Wallets except in import/export use cases and watchonly wallets.
+
+#### Creating Descriptor Wallets
+
+Descriptor Wallets are not created by default. They must be explicitly created using the
+`createwallet` RPC or via the GUI. A `descriptors` option has been added to `createwallet`.
+Setting `descriptors` to `true` will create a Descriptor Wallet instead of a Legacy Wallet.
+
+In the GUI, a checkbox has been added to the Create Wallet Dialog do indicate that a
+Descriptor Wallet should be created.
+
+Without those options being set, a Legacy Wallet will be created instead. Additionaly the
+Default Wallet created upon first startup of Bitcoin Core will be a Legacy Wallet.
+
+#### `IsMine` Semantics
+
+`IsMine` refers to the function used to determine whether a script belongs to the wallet.
+This is then used to determine whether an output belongs to the wallet. Legacy Wallets
+have a confusing definition of `IsMine` due to using keys. Keys can be involved in a variety
+of different scripts and many of those scripts are considered part of the wallet. This results
+in scripts being part of the wallet that the user is not necessarily aware of.
+
+With Descriptor Wallets, descriptors explicitly specify the set of scripts that are owned by
+the wallet. Since descriptors are deterministic and easily enumerable, users will know exactly
+what scripts the wallet will consider to belong to it. Additionally the implementation of `IsMine`
+in Descriptor Wallets is far simpler than for Legacy Wallets. Notably, in Legacy Wallets, `IsMine`
+allowed for users to take on type of address (e.g. P2PKH), mutate it into another address type
```

Comment:
"take on" -> "take in the" ?

Reply:
It was supposed to be "take one type of address". fixed